CODE_OF_CONDUCT.md

```diff
@@ -4,6 +4,6 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 
 Resources:
 
-- [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/)
-- [Microsoft Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/)
-- Contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with questions or concerns
+-   [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/)
```

Comment:
three spaces?

Reply:
That's prettier's opinion on markdown bullets. I don't recall why this was the policy they chose, but it is 🤷‍♂️